index.html

```diff
@@ -1,5 +1,5 @@
 <!doctype html>
-<html>
+<html lang="en">
```

Comment:
Adding the lang helps non-human readers to identify what language is to be expected from this page.

https://developer.mozilla.org/en-US/docs/Web/HTML/Global_attributes/lang